build-configs/configs/BuildConfigType.js

```diff
@@ -8,6 +8,34 @@ export type FeatureFlagsType = {|
   introSlides: boolean
 |}
 
+type AndroidGoogleServicesConfig = {|
+  google_app_id: string,
+  gcm_defaultSenderId: string,
+  default_web_client_id: string,
+  ga_trackingId: null | string,
+  firebase_database_url: string,
+  google_api_key: string,
+  google_crash_reporting_api_key: string,
+  project_id: string
```

Comment:
That's how these variables are used in the documentation of Google Services. I can refactor this and rename them. This could potentially make it more difficult to get what happens here.

What do you think?

Reply:
Hmm... I just find it strange that everything is camelCase except the firebase stuff. Not sure if it is worth the hassle though.